constructorio-client/src/main/java/io/constructor/client/models/RefinedContent.java

```diff
@@ -0,0 +1,17 @@
+package io.constructor.client.models;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.Map;
+
+public class RefinedContent {
+
+  @SerializedName("data")
+  private Map<String, String> data;
+
+  /**
+   * @return refined content data
+   */
+  public Map<String, String> getData() {
```

Comment:
I think the dashboard always adds the data as <String, String> but the end user has the option to add arbitrary JSON data through the API according to [this](https://docs.constructor.io/rest_api/search/refined_content_structure/). Should we consider <String, Object> here?

Reply:
Just checked by updating an existing rule and that seems to be the case. Will update, thanks for reviewing! ❤️ 

<img width="914" alt="Screen Shot 2022-09-23 at 5 07 19 PM" src="https://user-images.githubusercontent.com/7194266/192071228-5a1430ec-3e52-44f0-bfc8-3197a3ae9d51.png">

Reply:
I updated it, but it kind of sucks that you can't really access the data in the nested data objects

For example, `response.getResponse().getRefinedContent().get(0).getData().get("arbitraryDataObject").get("pizza");` doesn't work since `.get("arbitraryDataObject")` gives you back an object which isn't super great, but I guess metadata works the same way, so... :shrug